app/scripts/services/dimBungieService.factory.js

```diff
@@ -385,7 +321,8 @@
           return $q.all([
             getDestinyInventories(data.token, platform, data.membershipId, data.characters),
             getDestinyProgression(data.token, platform, data.membershipId, data.characters),
-            getDestinyAdvisors(data.token, platform, data.membershipId, data.characters)
+            getDestinyAdvisors(data.token, platform, data.membershipId, data.characters),
+            getDestinyVendors(data.token, platform, data.membershipId, data.characters)
```

Comment:
I wish this weren't tied to the stores load - if we could delay loading these until we go to the vendor page it'd be way faster.